### PR TITLE
New package: Gekorm.Dart.stable version 1.6.0

### PR DESCRIPTION
**Pull request by TejasMate**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/126671)

### CODE REVIEW

**Comment by BrandonWanHuanSheng** [comment]
While I am in sorting the string file.

**Comment by BrandonWanHuanSheng** [comment]
The version is inconsistent in the manifest which cause upgrade loops.

File: manifests/g/Gekorm/Dart/stable/1.6.0/Gekorm.Dart.stable.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Gekorm.Dart.stable
+PackageVersion: 1.6.0
+InstallerType: burn
```
Comment (BrandonWanHuanSheng) [inline manifests/g/Gekorm/Dart/stable/1.6.0/Gekorm.Dart.stable.installer.yaml:6]:
```suggestion
InstallerType: inno
InstallerSwitches:
   Silent: /SILENT
   Custom: /NORESTART /SUPPRESSMSGBOXES
```

**Review by BrandonWanHuanSheng — CHANGES_REQUESTED** [review]
This setup type is inno  It is not burn.

**Comment by stephengillie** [comment]
> This pull request unable to merge because it was end in a upgrade loops.

It appears that the problematic version was [removed](https://github.com/microsoft/winget-pkgs/pull/126672). This PR seems like an attempt to readd the version in a non-problematic way.

**Comment by stephengillie** [comment]
After install, I tested `winget upgrade` and it found no upgrade packages. So I believe there won't be an upgradae loop by approving this PR. 

```
PS C:\Users\User\Desktop> &"C:\Program Files\Dart\dart-sdk\bin\dart.exe"
A command-line utility for Dart development.

Usage: dart <command|dart-file> [arguments]

Global options:
-v, --verbose               Show additional command output.
    --version               Print the Dart SDK version.
    --enable-analytics      Enable analytics.
    --disable-analytics     Disable analytics.
    --suppress-analytics    Disallow analytics for this `dart *` run without changing the analytics configuration.
-h, --help                  Print this usage information.

Available commands:
  analyze    Analyze Dart code in a directory.
  compile    Compile Dart to various formats.
  create     Create a new Dart project.
  devtools   Open DevTools (optionally connecting to an existing application).
  doc        Generate API documentation for Dart projects.
  fix        Apply automated fixes to Dart source code.
  format     Idiomatically format Dart source code.
  info       Show diagnostic information about the installed tooling.
  pub        Work with packages.
  run        Run a Dart program.
  test       Run tests for a project.

Run "dart help <command>" for more information about a command.
See https://dart.dev/tools/dart-tool for detailed documentation.
PS C:\Users\User\Desktop> winget upgrade
No installed package found matching input criteria.
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**